vault/logical_system_test.go

```diff
@@ -3290,12 +3306,19 @@ func TestSystemBackend_ToolsHash(t *testing.T) {
 
 func TestSystemBackend_ToolsRandom(t *testing.T) {
 	b := testSystemBackend(t)
+	paths := b.(*SystemBackend).toolsPaths()
 	req := logical.TestRequest(t, logical.UpdateOperation, "tools/random")
 
-	_, err := b.HandleRequest(namespace.RootContext(nil), req)
+	resp, err := b.HandleRequest(namespace.RootContext(nil), req)
 	if err != nil {
 		t.Fatalf("err: %v", err)
 	}
+	schema.ValidateResponse(
+		t,
+		schema.FindResponseSchema(t, paths, 1, req.Operation),
```

Comment:
If it's not too much effort, could you please convert these to `GetResponseSchema(t, b.Route(...), ...)`

Reply:
will do! thought i got em all